File: Python/Condicional/Intermedio/Ejercicio_04.py
# Solicitar al usuario ingresar las horas trabajadas
horas_trabajadas = float(input("Ingrese las horas trabajadas: "))

# Definir salario por hora normal
salario_normal = 10.0  # Reemplaza con el valor real

# Calcular el salario según las horas extras
if horas_trabajadas > 40:
    horas_normales = 40
    horas_extras = horas_trabajadas - 40
    salario = (horas_normales * salario_normal) + (horas_extras * salario_normal * 2)
else:
    salario = horas_trabajadas * salario_normal

print("El salario es:", salario)

File: Java/src/main/java/org/sisuns/condicionales/intermedio/Ejercicio_04.java
package org.sisuns.condicionales.intermedio;

public class Ejercicio_04 {
    public static void main(String[] args) {
        // Definir el salario normal por hora (asignar un valor específico)
        double salarioNormal = 10.0;

        // Solicitar al usuario ingresar las horas trabajadas
        System.out.print("Ingrese las horas trabajadas: ");
        int horasTrabajadas = 0; // Puedes asignar un valor específico si lo deseas
        // Leer las horas trabajadas
        // horasTrabajadas = ...; // Puedes utilizar Scanner si lo prefieres

        // Calcular el salario según las horas extras
        double salario;
        if (horasTrabajadas > 40) {
            int horasNormales = 40;
            int horasExtras = horasTrabajadas - 40;
            salario = (horasNormales * salarioNormal) + (horasExtras * salarioNormal * 2);
        } else {
            salario = horasTrabajadas * salarioNormal;
        }

        // Mostrar el salario
        System.out.println("El salario es: " + salario);
    }
}
